birdspotter/birdspotter/accounts/views.py

```diff
@@ -2,14 +2,17 @@
 from django.contrib.admin.views.decorators import staff_member_required
 from django.contrib.auth import authenticate, login, logout
 from django.contrib.auth.decorators import login_required
+from django.core.mail import send_mail
```

Comment:
Please remove this unused import ( I would, but then we'd need a third person to review it)